pom.xml

```diff
@@ -55,4 +55,23 @@
       <url>http://repo.jenkins-ci.org/public/</url>
     </pluginRepository>
   </pluginRepositories>
+    
+    <dependencies>
+        
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.hamcrest</groupId>
+            <artifactId>hamcrest-all</artifactId>
+            <version>1.3</version>
+            <scope>test</scope>
+        </dependency>

```

Comment:
It may be temporary bug with my IDE, but it can't find `Matchers.hasSize()` matcher. Firstly i've think that it not included to non-lts release, but now i don't know. Will check it once more with clean pull.